doc/api/process.md

```diff
@@ -1730,6 +1730,70 @@ function definitelyAsync(arg, cb) {
 }
 ```
 
+### When to use `queueMicrotask()` vs. `process.nextTick()`
+
+The [`queueMicrotask()`][] API is an alternative to `process.nextTick()` that
```

Comment:
I feel like it would be important to state that it is "the portable alternative to Node.js' specific `process.nextTick()`."

Generally I am wondering if this wording may sound a little bit easier to read:

> The [`queueMicrotask()`][] API is the portable alternative to `process.nextTick()`. It defers execution of a function using the same as the microtask queue as the then, catch and finally handlers of resolved promises. Within Node.js the microtask queue is drained immediately after the "next tick queue".

Reply:
But they are not the same thing, so calling it an alternative is a bit confusing, no?

Reply:
I don't think so. They are both alternatives for deferring execution of a function, they just do so in oh so slightly different ways. Calling them alternatives is fine so long as we detail what those caveats are, which we do.